ROSCO_toolbox/turbine.py

```diff
@@ -607,7 +607,11 @@ def __init__(self,performance_table, pitch_initial_rad, TSR_initial):
         # --- Find TSR ---
         # Make finer mesh for Tip speed ratios at "optimal" blade pitch angle, do a simple lookup. 
         #       -- nja: this seems to work a little better than interpolating
-        performance_beta_max = np.ndarray.flatten(performance_table[:,self.max_ind[1]]) # performance metric at maximizing pitch angle
+        if len(self.max_ind)==1:
```

Comment:
I think `self.max_ind` normally has two dimensions and this should be `self.max_ind[1]`.  Pushing a change now.

Reply:
Yes, you are right